tests/Unit/Helpers/Domain/DomainTestHelpers.cpp

```diff
@@ -255,6 +257,70 @@ tnsr::I<double, VolumeDim, Frame::BlockLogical> point_in_neighbor_frame(
   return get_corner_of_orthant(point_get_orthant);
 }
 
+template <size_t VolumeDim>
+void check_block_face_grid_points_align(
+    const Block<VolumeDim>& block1, const Block<VolumeDim>& block2,
+    double time = std::numeric_limits<double>::signaling_NaN(),
+    const std::unordered_map<
+        std::string, std::unique_ptr<domain::FunctionsOfTime::FunctionOfTime>>&
+        functions_of_time = {}) {
+  const auto direction = find_direction_to_neighbor(block1, block2);
+  const auto orientation = find_neighbor_orientation(block1, block2);
+  // corner points are checked by physical_separation, so check on gauss points
+  const Mesh<VolumeDim - 1> face_mesh{3_st, Spectral::Basis::Legendre,
+                                      Spectral::Quadrature::Gauss};
+  tnsr::I<DataVector, VolumeDim, Frame::ElementLogical> xi =
```

Comment:
I stared at this for a while to figure out what the code is doing, could you add a comment above this line explaining a bit what's going on?